google-cloud-storage/src/test/resources/META-INF/native-image/com/google/cloud/storage/resource-config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources":{
+    "includes":[
+      {"pattern": ".*.txt"}
```

Comment:
Is it bad if we include everything under `src/test/resources/`? That would match more the expectation of the full classpath being included.

Reply:
It might be okay to move this under src/test/resources. Are we referencing any `.txt` files in the regular application?

Reply:
No, we currently don't have anything under `src/main/resources/` that would need to ship in the jar or be taken into consideration for native-image.

Reply:
Nice, keeping this in `src/test/resources` in that case.